R/calculate_recommended_interventions.R

```diff
@@ -1,123 +1,170 @@
 #' calculate_recommended_interventions
 #'
-#' @description Returns the LAGO recommended interventions for the next stage.
-#' The recommended interventions that aim to satisfy the outcome goal and/or the
+#' @description Calculates the LAGO recommended interventions for the next stage.
+#' The recommended interventions that aim to satisfy an outcome goal and/or a
 #' power goal.
 #'
-#' @param df data.frame, Input data
-#' @param outcome_name character, Outcome name
-#' @param outcome_type character, Outcome type, either "continuous" or "binary"
-#' @param glm_family character, Family for the GLM model
-#' For example: "binomial"
-#' For v1.1, we are only considering binomial family, and quasibinomial(link="logit")
-#' @param link character, the link option when glm_family is set to quasibinomial.
-#' It has to be one of the following:
-#' logit (default), probit, cauchit, log and cloglog.
-#' @param include_intercept boolean, Whether the intercept should be included in the
-#' fitted model.
-#' @param interventions_list character vector, Names of the intervention components
-#' For example: c("component1", "component2")
-#' @param center_characteristic_list character vector, Names of the center characteristics
-#' For example: c("characteristic1")
-#' @param center_characteristic_list_for_optimization numeric vector, Given values of
-#' the center characteristics
+#'
+#' ## required arguments:
+#' @param data data.frame: input data frame.
```

Comment:
Maybe a bit of rewriting to make it clearer? Before the way you wrote it data data.frame in the same line got me confused at the first glance because it thought those two are one word, not one word and its input type. My suggestions:
#' @param data A data.frame. The input dataset containing the variables of interest.
#' @param outcome_name A character string. The name of the column in the dataset that represents the outcome of interest.
#' @param outcome_type A character string. Specifies the type of the outcome. Must be either "continuous" for numeric outcomes or "binary" for categorical outcomes.

Reply:
agreed

Reply:
I mistyped. My suggestions were:
#' @param data A `data.frame`. The input dataset containing the variables of interest.
#' @param outcome_name A `character` string. The name of the column in the dataset that represents the outcome of interest.
#' @param outcome_type A `character` string. Specifies the type of the outcome. Must be either "continuous" for numeric outcomes or "binary" for categorical outcomes.
And this applies for the rest of the parameter specifications.